packages/config-array/tests/config-array.test.js

```diff
@@ -10,13 +10,14 @@
 import { ConfigArray, ConfigArraySymbol } from "../src/config-array.js";
 import path from "node:path";
 import assert from "node:assert";
+import { fileURLToPath } from "node:url";
 
 //-----------------------------------------------------------------------------
 // Helpers
 //-----------------------------------------------------------------------------
 
 // calculate base path using import.meta
-const basePath = path.dirname(new URL(import.meta.url).pathname);
```

Comment:
On Windows, this was giving a path like `/c:/foo/bar/rewrite/packages/config-array/tests`, where the `c:` is considered a top-level directory rather than a drive specifier. This was producing incorrect results with `path.resolve(basePath)` like `C:\c:\foo\bar\rewrite\packages\config-array\tests`.